src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1335,14 +1355,55 @@ export default class AircraftModel {
         this.setFlightPhase(FLIGHT_PHASE.WAITING);
     }
 
+    /**
+     * Initialize all autopilot systems after being given an IFR clearance to destination and execute takeoff.
+     *
+     * @for Pilot
+     * @method takeoff
+     * @param runway {RunwayModel} the runway taking off on
+     * @param initialAltitude {number} the altitude aircraft can automatically climb to at this airport
+     */
+    takeoff(runway, initialAltitude) {
```

Comment:
moved here from Pilot, so that we can remove the Pilot in a distanced future. But I think we have to split AircraftModel sooner or later or we end up with an unmaintainable file.

Reply:
Good question. No idea. One thing that I have already done in this PR is to move scoring/procedure related code out of AircraftModel. I think there is probably more code that could be extracted (collision, airspace bust). 

Maybe we could have a pilot class that controls the aircraft (the machine) and an aircraft class that represents the aircraft as a machine. But I haven't really thought about that. Maybe we should also have a flight/flight plan class for things like flight number, origin, and destination.